docs/content/contribute/index.md

```diff
@@ -10,6 +10,7 @@ Thanks for your interest in contributing to HyperShift. Here are some guidelines
 * For bigger changes (more than 200 lines of code diff), do not just do the change but ask for feedback on the idea and direction of the change first (Either in a GitHub issue or the #project-hypershift channel in the CoreOS slack). This avoids situations where big changes are submitted that are then declined or never reviewed, which is frustrating for everyone.
 * Regardless of the size of the change, always explain how the change will improve the project.
 * Make sure the "Why" and "How" are included in the message of each commit.
+* Use the imperative mood in the subject line for every commit.
```

Comment:
s/mood/mode

Reply:
mood is a thing :) ... see link above

Reply:
Oops! Thanks. I guess I was in the undercaffeinated mood...

Reply:
yup, for more references in addition to Cesar's link
https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=HEAD#n145
https://initialcommit.com/blog/Git-Commit-Message-Imperative-Mood

Reply:
We should probably make `imperative mood` a link, it seems it is not a well-known term. I was aware of the underlying concept but I've never head the term before

Reply:
Added the link.